tools/llm_bench/llm_bench_utils/config_class.py

```diff
@@ -62,6 +62,7 @@
         't5',
         'falcon',
         "glm",
+        "gpt",
         'gpt-',
```

Comment:
if we have `gpt`, do we need also `gpt-` and `gpt2` ?

Reply:
this is needed for backward compatibility. I have additional task for cleanup configuration types list, it will be done separately (this onle only for unblocking validation teams)